Assets/MRTK/Core/Providers/Hands/HandBounds.cs

```diff
@@ -14,9 +14,14 @@ namespace Microsoft.MixedReality.Toolkit.Input
     public class HandBounds : MonoBehaviour, IMixedRealitySourceStateHandler, IMixedRealityHandJointHandler
     {
         /// <summary>
-        /// Accessor for the bounds associated with a handedness.
+        /// Accessor for the bounds associated with a handedness, calculated in global-axis-aligned space.
         /// </summary>
         public Dictionary<Handedness, Bounds> Bounds { get; private set; } = new Dictionary<Handedness, Bounds>();
+        
+        /// <summary>
+        /// Accessor for the bounds associated with a handedness, calculated in local hand-space, locally axis aligned.
+        /// </summary>
+        public Dictionary<Handedness, Bounds> LocalBounds { get; private set; } = new Dictionary<Handedness, Bounds>();
```

Comment:
Could you remove the hand OnSourceLost like we do for the global bounds?

Reply:
nit: Might be cool to have an option to draw the local bounds too like we do for global bounds via the drawBoundsGizmo property. Should be pretty easy to do by setting the gizmo matrix.

Reply:
>Could you remove the hand OnSourceLost like we do for the global bounds?

Wow thank you for spotting that. That was a big oversight on my part. Thank you.

Didn't know about gizmo matrices. That is a cool feature; I had sort of assumed that gizmos were just "you get what you get". I will add that.

Reply:
Gizmo improvements now added.